onedrive_service/src/onedrive_service/service.py

```diff
@@ -0,0 +1,182 @@
+""" Provides service to interact with remote API
+"""
+# pylint: disable=too-many-arguments
+import onedrivesdk
+
+
+class OneDriveService(object):
```

Comment:
The name for this class was changed, please refer to:
https://github.com/epam/OneDrive-L/pull/16

Reply:
OneDriveUtil